bbinc/cdb2_constants.h

```diff
@@ -19,7 +19,7 @@
 
 #define COMDB2_MAX_RECORD_SIZE 16384
 #define LONG_REQMS 2000
-#define MAXBLOBLENGTH 256 * 1024 * 1024 /* TODO: set a good maximum here */
+#define MAXBLOBLENGTH 255 * 1024 * 1024 /* TODO: set a good maximum here */
```

Comment:
Comdb2sys [expected output](https://github.com/bloomberg/comdb2/blob/master/tests/comdb2sys.test/comdb2sys.expected#L277) needs to be adjusted.